Libraries/Opc.Ua.Client/Opc.Ua.Client.csproj

```diff
@@ -22,7 +22,12 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' != 'net6.0' AND '$(TargetFramework)' != 'net7.0'">
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="6.0.1" />
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.6.0" />
+    <PackageReference Include="OpenTelemetry.Api" Version="1.6.0" />
   </ItemGroup>
```

Comment:
why is open telemetry needed?

Reply:
OpenTelemetry is needed to have access the propgagator and get baggage from the PropagationContext

Reply:
removed the dependency